pkg/workload/tpcc/tpcc.go

```diff
@@ -137,6 +141,61 @@ func (w *waitSetter) String() string {
 	}
 }
 
+var _ pgx.QueryTracer = fileLoggerQueryTracer{}
+
+type fileLoggerQueryTracer struct {
+	filePath string
+	file     *os.File
+}
+
+func newFileLoggerQueryTracer(filePath string) (*fileLoggerQueryTracer, error) {
+	tracer := &fileLoggerQueryTracer{filePath: filePath}
+	file, err := os.OpenFile(tracer.filePath, os.O_CREATE|os.O_TRUNC|os.O_RDWR, 0666)
```

Comment:
Nit: Are we sure about truncating the file silently? IMO, a friendlier default is to fail if the file already exists, but maybe that's just me.

Reply:
Yup that sounds like a better idea to me